src/screens/Home/Home.tsx

```diff
@@ -12,9 +13,17 @@ const styles = StyleSheet.create({
 });
 
 export default function Home() {
+  const { navigate } = useNavigation();
```

Comment:
```suggestion
export default function Home({ navigation }: RootStackScreenProps<'Home'>) {
  const { navigate } = navigation;
```

Better to use it like this in screens, right?

Reply:
Updated